media/juxtapose/css/playhead.css

```diff
@@ -4,6 +4,9 @@
 .jux-playhead-container input[type=range] {
     cursor: col-resize;
     width: 930px;
+    -webkit-appearance: none;
```

Comment:
Oh maybe there's a way that involves something trickier used in that post - I'll look into it.